### DISCUSSION TITLE
GitOpsCon EU 2023 as a Virtual Event

### DISCUSSION THREAD

**Discussion by christianh814** [open-gitops/project#164] [Ideas]
The idea was passed around to do "GitOpsCon EU 2023" a virtual event. The reason being is that OpenSource Summit EU is in Sept and there will just not be enough time for planning. [We have done this in the past with Red Hat letting us use their HopIn account for the event.](https://hopin.com/events/gitops-con)

The idea being that this should be a "stop gap" until the LF/CNCF comes back with their plans for the event.

Emoji vote (+comment too!)

👍🏽 - Yes to a virtual event for the EU
👎🏽 - No to a virtual event for the EU

Outstanding Questions:

- Should this be a "free" event?
- Should we go through LF/CNCF for this virtual event as well?
- Should we engage with Red Hat again to see if we can use Hopin? (i.e. it'll be "free for us")
- Should we have multiday/multitrack? Single Day/Single track? Some other combination?
- Other considerations? (Add your comment)

**Comment by christianh814** [comment]
My thoughts would be to make this a free event, single day/dual track, during EU hours (Maybe 1000-1600 Central European Timezone). The Red Hat sponsorship of letting us use Hopin is a good idea, because I'm not sure what else we can do that is "free"; but I'm open to suggestions (maybe LF/CNCF can help here too?)

The communication of this should be that it is a "stop gap" with a more official, in person, EU event to happen in 2024.

**Reply by niklasmtj** [reply]
Yes to all your points. 

One thing that went through my mind about the times is trying to find a slot which is "early" but still interesting for US folks and "late" but still interesting for asian timezones. What I thought was a two day event from maybe 12:00-16:00. This way we can even do more presentations / demos etc. and have a time where possibly more people from around the world are able to participate. 

And yes, I would like to see an official, in person, GitOpsCon in EU next year.

**Comment by niklasmtj** [comment]
Hosting this event is a big question. I've never done or organized an online event. 

Having the option to use Hopin or something from LF/CNCF would be great. I think it depends on what we would like to do with the event. Doing a "classic" event with sponsor "virtual tables/expo area" or just one/two tracks (read streams).

**Comment by staceypotter** [comment]
Can we close this discussion please and thank you?
